### PR TITLE
Assets page redesign

### PR DESCRIPTION
**Pull request by kattylucy**
- Redesign to asset overview page & asset detail page

#2507 

- [ ] Dev

### CODE REVIEW

File: centrifuge-app/src/components/AssetSummary.tsx [hunk 1]
```diff
@@ -16,26 +15,21 @@ type Props = {
...
-          boxShadow: `0 1px 0 ${theme.colors.borderPrimary}`,
-        }}
-      >
+    <Stack
+      bg={theme.colors.backgroundSecondary}
+      border={`1px solid ${theme.colors.borderSecondary}`}
+      borderRadius={10}
+      padding={2}
+      marginLeft={[2, 2, 2, 2, 5]}
+      marginRight={[2, 2, 2, 2, 5]}
```
Comment (onnovisser) [inline centrifuge-app/src/components/AssetSummary.tsx:24]:
```suggestion
      mx={[2, 2, 2, 2, 5]}
```

File: centrifuge-app/src/components/Charts/AssetPerformanceChart.tsx [hunk 2]
```diff
@@ -249,22 +230,22 @@ function AssetPerformanceChart({ pool, poolId, loanId }: Props) {
                           {payload.map(({ value }, index) => (
                             <>
                               <Shelf justifyContent="space-between" pl="4px" key={index}>
-                                <Text variant="label2">{'Value'}</Text>
-                                <Text variant="label2">
+                                <Text variant="body3">{'Value'}</Text>
+                                <Text variant="body3">
                                   {payload[0].payload.historicPV
-                                    ? formatBalance(payload[0].payload.historicPV, 'USD', 2)
+                                    ? formatBalance(payload[0].payload.historicPV, pool.currency.name, 2)
```
Comment (onnovisser) [inline centrifuge-app/src/components/Charts/AssetPerformanceChart.tsx:236]: should be `pool.currency.symbol` for all of them

File: centrifuge-app/src/components/PageSection.tsx [hunk 3]
```diff
@@ -43,17 +43,7 @@ export function PageSection({
...
-      pl={3}
-      pr={[3, 3, 7]}
-      pt={3}
-      pb={4}
-      gap={3}
-      borderTopWidth={1}
-      borderTopStyle="solid"
-      borderTopColor="borderPrimary"
-    >
+    <Stack as="section" padding={2} marginLeft={[0, 0, 0, 1, 3]} marginRight={[0, 0, 0, 1, 3]} gap={3}>
```
Comment (onnovisser) [inline centrifuge-app/src/components/PageSection.tsx:46]:
```suggestion
    <Stack as="section" padding={2} mx={[0, 0, 0, 1, 3]} gap={3}>
```

File: centrifuge-app/src/pages/Loan/ExternalRepayForm.tsx [hunk 4]
```diff
@@ -187,94 +198,101 @@ export function ExternalRepayForm({ loan, destination }: { loan: ExternalLoan; d
   return (
     <FormikProvider value={repayForm}>
       <Stack as={Form} gap={3} noValidate ref={repayFormRef}>
-        <Stack gap={1}>
-          <Shelf gap={1}>
+        <Box
+          padding="24px 16px"
```
Comment (onnovisser) [inline centrifuge-app/src/pages/Loan/ExternalRepayForm.tsx:202]:
```suggestion
          px={3}
          py={2}
```
prefer theme values over hardcoded ones

File: centrifuge-app/src/pages/Pool/Assets/OffchainMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  background: transparent;
+  border: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  font-family: Inter, sans-serif;
+`
+
+const LoanOption: React.FC<LoanOptionProps> = ({ loan }) => {
```
Comment (onnovisser) [inline centrifuge-app/src/pages/Pool/Assets/OffchainMenu.tsx:38]: `React.FC` is obsolete. Normal functions will do

File: fabric/src/components/InlineFeedback/index.tsx [hunk 6]
```diff
@@ -26,7 +26,7 @@ export function InlineFeedback({ status = 'default', children }: InlineFeedbackP
     <Text variant="body3">
       <Shelf alignItems="baseline" gap="4px">
         <StyledIconWrapper minWidth="iconSmall" height="iconSmall" flex="0 0 auto">
-          <StyledIcon as={icons[status]} size="iconSmall" color={`status${capitalizeFirstLetter(status)}`} />
+          <StyledIcon as={icons[status]} size={24} color={`status${capitalizeFirstLetter(status)}`} />
```
Comment (onnovisser) [inline fabric/src/components/InlineFeedback/index.tsx:29]:
```suggestion
          <StyledIcon as={icons[status]} size="iconMedium" color={`status${capitalizeFirstLetter(status)}`} />
```

**Comment by onnovisser** [comment]
Looking really nice!

**Review by onnovisser — DISMISSED** [review]

File: centrifuge-app/src/components/LoanList.tsx [hunk 7]
```diff
@@ -157,98 +177,141 @@ export function LoanList({ loans }: Props) {
...
+        'Maturity Date': loan.maturityDate ? loan.maturityDate : '-',
+        Quantity: `${quantity ?? '-'}`,
+        'Market Price': loan.marketPrice ? loan.marketPrice : '-',
+        'Market Value': loan.marketValue ? loan.marketValue : '-',
+        'Unrealized P&L': loan.unrealizedPL ? loan.unrealizedPL : '-',
+        'Realized P&L': loan.realizedPL ? loan.realizedPL : '-',
+        'Portfolio %': loan.portfolioPercentage ? loan.portfolioPercentage : '-',
+      }
+    })
+  }, [rows])
```
Comment (sophialittlejohn) [inline centrifuge-app/src/components/LoanList.tsx:263]: would add the suggested dependency `pool` here

File: centrifuge-app/src/components/Tooltips.tsx [hunk 8]
```diff
@@ -342,6 +342,10 @@ export const tooltipText = {
     label: 'Expense ratio',
     body: 'The operating expenses of the fund as a percentage of the total NAV',
   },
+  totalNavMinus: {
```
Comment (sophialittlejohn) [inline centrifuge-app/src/components/Tooltips.tsx:345]:
```suggestion
  totalNavMinusFees: {
```

**Review by sophialittlejohn — APPROVED** [review]
Nice work 🔥 🔥 🔥  besides the table sorting now working right yet I think it is good enough to ship 🚢 proper table sorting can be handled in a follow up PR if it will block the release

**Comment by hieronx** [comment]
@kattylucy Looking great! One thing I noticed: there is now a new asset with 0 value. The market price shows up as `USDC` which should be `0 USDC`, and likewise for both P&L columns.

Screenshot 2024-11-06 at 09 02 27

I also asked Brian to come up with a loading state: https://www.figma.com/design/ng7qdNcSCXSDA6ZUdWIs6u/Pool-Overview%2F-Pool-Detail?node-id=4516-2623&t=DshORdTnFD448TSr-4 This would be nice to implement while the data in the table is loading (and a pattern we should start adopting across the app). But consider this a very nice-to-have, happy to write a separate ticket for this as well.

**Comment by hieronx** [comment]
Also one smaller point, the download icon is off:

Screenshot 2024-11-06 at 09 06 08

Compared to design:

Screenshot 2024-11-06 at 09 06 22

**Pull request merged**